templates/internet-of-things/appstore.html

```diff
@@ -11,7 +11,7 @@
   <div class="p-strip--suru-half-and-half">
     <div class="row u-equal-height">
       <div class="col-7 col-medium-4">
-        <h1>White-label app stores<br class="u-hide--small" />for IoT</h1>
+        <h1>White-label app stores<br class="u-hide--small" />for Linux and IoT</h1>
```

Comment:
You need a space before the `br` otherwise on small screens the words "stores" and "for" are spaceless.